docs/extending-minerva.rst

```diff
@@ -0,0 +1,68 @@
+Extending Minerva
+=================
+
+Creating a Minerva Plugin
```

Comment:
I'm a little confused by this whole section.  Maybe we can define what a "Minerva Plugin" is?

A bit later you say "Minerva specific plugins", so is that different from a "Minerva Plugin"?

I think because some of this stuff is tricky, it is better to err on the side of being overly explicit.


Maybe the title can be "Creating a Minerva plugin through the Girder plugin system"?

and then

"For Minerva specific plugins" => "For Girder plugins intended to extend the functionality of Minerva"?